x-pack/metricbeat/module/openmetrics/collector/manifest.yml

```diff
@@ -0,0 +1,10 @@
+default: true
+input:
+  module: prometheus
+  metricset: collector
```

Comment:
I like the suggestion of replacing `openmetrics.collector` with `openmetrics.text` since it makes it more future proof.

Reply:
I think protocol is an implementation detail that can be handled by the metricset itself right? This would probably be auto detected from the HTTP envelope. We could just add a parameter to force the protocol if needed, but I don't really see the need for 2 different metricsets

Reply:
Ok, let's keep it as a single metricset and we can handle the protocol there as we do in other cases.